yarn.lock

```diff
@@ -1607,6 +1607,24 @@
     lodash "^4.17.11"
     moment "^2.24.0"
 
+"@ovh-ux/manager-pci@^0.5.0-alpha.0":
```

Comment:
`yarn install` from `96c54bf94b4f677ce080169aa6c64ec4633b1706` causes this following change.

Reply:
Thanks for the explanation